design-system-templates/example-pages/landing.html.twig

```diff
@@ -14,7 +14,7 @@
 	{% include 'components/hero-landing.html.twig' %}
 
 	<div class="component component--columns component--columns--images">
-		<div class="component--columns__intro">
+		<div>
```

Comment:
Same comment: Is that now class-less div necessary?

Reply:
Removed in latest commits